VirtoCommerce.CatalogModule.Web.Core/Model/MoveContext.cs

```diff
@@ -1,18 +1,13 @@
-﻿using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Web;
 
 namespace VirtoCommerce.CatalogModule.Web.Model
 {
-    /// <summary>
-    /// Represent move operation detail
-    /// </summary>
-    public class MoveInfo
+    public class MoveContext
```

Comment:
I think we should not rename existing classes, as it brakes backward compatibility. And in general I do not see the real difference between MoveInfo and MoveContext.
Could you just rename the file to MoveInfo?